### PR TITLE
Add Work section, About Myself section

### PR DESCRIPTION
**Pull request by dev89s**
1. Added the "Works section" with many cards representing different projects done in the past.
2. Added the "About Myself" section including a brief description, social network links, and a link to get the Resume.
3. Added a list of different languages and frameworks that the owner is familiar with and also a list of different skills he has.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,6 +57,149 @@ <h3 class="connect">LET'S CONNECT</h3>
...
         </ul>
+        <div class="bottomspace"></div>
+      </section>
+
+      <!-- Works Section -->
+      <section class="works">
+
+        <!-- card 1 -->
+        <div class="card card1">
+          <img class="card-cover" src="src/images/card-cover1.png" alt="">
```
Comment (TadesseDev) [inline index.html:68]: - [ ] Your work section is looking great already, I just have one suggestion here. I notice your margin-top is a bit much compared with the figma design. Please minimize the space between the heading and the work section to make your design even better. 

Screenshot 2023-02-17 200924

File: style.css [hunk 2]
```diff
@@ -80,3 +97,191 @@ main {
...
+  font-size: 17px;
+}
+
+.resume-link:hover {
+  background-color: #6070ff;
+  color: #fff;
+  box-shadow:
+    1px 1px 5px rgba(0, 0, 0, 0.3),
+    -1px -1px 5px rgba(0, 0, 0, 0.3);
+}
```
Comment (TadesseDev) [inline style.css]: - [ ] Dear @dev89s, It's great you have a hover effect and it looks great. But please notice we are expected to add a `Active` and `Pressed` effect also. This goes to all other buttons too.

Screenshot 2023-02-17 220425

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @dev89s 
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @dev89s, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**